examples/run/run.cpp

```diff
@@ -975,9 +975,16 @@ static int generate(LlamaData & llama_data, const std::string & prompt, std::str
     printf(LOG_COL_DEFAULT);
     return 0;
 }
+static const char *default_prompt_prefix = "> ";
+static const char *prompt_prefix = nullptr;
+
+static void initialize_prompt_prefix() {
+    const char *llama_prompt_prefix = std::getenv("LLAMA_PROMPT_PREFIX");
+    prompt_prefix = (llama_prompt_prefix != nullptr) ? llama_prompt_prefix : default_prompt_prefix;
+}
+
 
 static int read_user_input(std::string & user_input) {
-    static const char * prompt_prefix = "> ";
```

Comment:
Lets simplify to:

```
    static const char * prompt_prefix_env = std::getenv("LLAMA_PROMPT_PREFIX");
    static const char * prompt_prefix     = prompt_prefix_env ? prompt_prefix_env : "> ";
```

no need for global scope variables, etc.

Reply:
ok, it was to avoid to read the value on each command being entered, I will amend

Reply:
That's what static does when declared in a function.

Reply:
amended